Dockerfile

```diff
@@ -1,14 +1,9 @@
 FROM nginx:alpine
 
-# Copy app files
-COPY build /usr/share/nginx/html
-# Install SPA-friendly config
-COPY nginx/default.conf /etc/nginx/conf.d/
-# Install JQ to work with JSON configs
-RUN apk add --no-cache jq
-
-#Configure and start the app
-WORKDIR /usr/share/nginx/html
-CMD /bin/ash -c "echo -n 'window.__CONFIG=' > config.js \
```

Comment:
Crap, I've never even heard of 'ash'.  I think Roman (who must have done this) was a bit of a contrarian ;-).